src/custom/pages/Claim/ClaimingStatus.tsx

```diff
@@ -1,17 +1,32 @@
 import { Trans } from '@lingui/macro'
-import { ConfirmOrLoadingWrapper, ConfirmedIcon, AttemptFooter, CowSpinner } from 'pages/Claim/styled'
+import {
+  ConfirmOrLoadingWrapper,
+  ConfirmedIcon,
+  AttemptFooter,
+  CowSpinner,
+  BannersWrapper,
+  SuccessBanner,
+} from 'pages/Claim/styled'
 import { ClaimStatus } from 'state/claim/actions'
 import { useClaimState } from 'state/claim/hooks'
 import { useActiveWeb3React } from 'hooks/web3'
 import CowProtocolLogo from 'components/CowProtocolLogo'
 import { useAllClaimingTransactions } from 'state/enhancedTransactions/hooks'
-import { useMemo } from 'react'
+import React, { useMemo } from 'react'
 import { Link } from 'react-router-dom'
 import { ExplorerLink } from 'components/ExplorerLink'
 import { EnhancedTransactionLink } from 'components/EnhancedTransactionLink'
 import { ExplorerDataType } from 'utils/getExplorerLink'
 import { V_COW } from 'constants/tokens'
 import AddToMetamask from 'components/AddToMetamask'
+import SVG from 'react-inlinesvg'
+import twitterImage from 'assets/cow-swap/twitter.svg'
+import discordImage from 'assets/cow-swap/discord.svg'
+import CowProtocolIcon from 'assets/cow-swap/cowprotocol.svg'
+import ExternalLink from 'theme'
```

Comment:
As David noted on slack, the fix is:

```suggestion
import { ExternalLink } from 'theme'
```